StreetSmartAPI/StreetSmart.WPF.Tests/SvgParameterCollectionTests.cs

```diff
@@ -0,0 +1,423 @@
+    Assert.NotNull(result);
+    Assert.Single(result.SvgParameter);
+    Assert.Equal(FillType.Fill, result.SvgParameter[0].Name);
+    Assert.Equal(expectedColorValue, result.SvgParameter[0].Value);
+    Assert.Equal(color.R, resultRed);
+    Assert.Equal(color.G, resultGreen);
+    Assert.Equal(color.B, resultBlue);
+  }
+
+  [Fact]
+  public void GetStrokeObject_OnlyRedColor_NullWidthAndOpacity_ReturnsExpectedResult()
+  {
+    // Arrange
+    Color color = Color.FromArgb(255, 255, 0, 0);
+    double? width = null;
+    double? opacity = null;
+    string expectedColorValue = "#FF0000";
+
+    // Act
+    var result = SvgParameterCollection<StrokeType>.GetStrokeObject(color, width, opacity);
+    var (resultRed, resultGreen, resultBlue) = HexToRgb(result.SvgParameter[0].Value);
+
+    // Assert
+    Assert.NotNull(result);
+    Assert.Single(result.SvgParameter);
+    Assert.Equal(StrokeType.Stroke, result.SvgParameter[0].Name);
+    Assert.Equal(expectedColorValue, result.SvgParameter[0].Value);
+    Assert.Equal(color.R, resultRed);
+    Assert.Equal(color.G, resultGreen);
+    Assert.Equal(color.B, resultBlue);
+  }
+
+  [Fact]
+  public void GetStrokeObject_OnlyGreenColor_NullWidthAndOpacity_ReturnsExpectedResult()
+  {
+    // Arrange
+    Color color = Color.FromArgb(255, 0, 255, 0);
+    double? width = null;
+    double? opacity = null;
+    string expectedColorValue = "#00FF00";
+
+    // Act
+    var result = SvgParameterCollection<StrokeType>.GetStrokeObject(color, width, opacity);
+    var (resultRed, resultGreen, resultBlue) = HexToRgb(result.SvgParameter[0].Value);
+
+    // Assert
+    Assert.NotNull(result);
+    Assert.Single(result.SvgParameter);
+    Assert.Equal(StrokeType.Stroke, result.SvgParameter[0].Name);
+    Assert.Equal(expectedColorValue, result.SvgParameter[0].Value);
+    Assert.Equal(color.R, resultRed);
+    Assert.Equal(color.G, resultGreen);
+    Assert.Equal(color.B, resultBlue);
+  }
+
+  [Fact]
+  public void GetStrokeObject_OnlyBlueColor_NullWidthAndOpacity_ReturnsExpectedResult()
+  {
+    // Arrange
+    Color color = Color.FromArgb(255, 0, 0, 255);
+    double? width = null;
+    double? opacity = null;
+    string expectedColorValue = "#0000FF";
+
+    // Act
+    var result = SvgParameterCollection<StrokeType>.GetStrokeObject(color, width, opacity);
+    var (resultRed, resultGreen, resultBlue) = HexToRgb(result.SvgParameter[0].Value);
+
+    // Assert
+    Assert.NotNull(result);
+    Assert.Single(result.SvgParameter);
+    Assert.Equal(StrokeType.Stroke, result.SvgParameter[0].Name);
+    Assert.Equal(expectedColorValue, result.SvgParameter[0].Value);
+    Assert.Equal(color.R, resultRed);
+    Assert.Equal(color.G, resultGreen);
+    Assert.Equal(color.B, resultBlue);
+  }
+
+
+  public static (int Red, int Green, int Blue) HexToRgb(string hex)
+  {
+    hex = hex.TrimStart('#');
+
+    int argb = int.Parse(hex, System.Globalization.NumberStyles.HexNumber);
+
+    int red = (argb >> 16) & 0xFF;
```

Comment:
Simple transformation, like this ColorTranslator.FromHtml(hex) can be used here.